server/views/pages/deactivateTemporary/details.njk

```diff
@@ -61,9 +61,54 @@
           {% endif %}
 
           {% block fields %}
+          
             {% for key, fieldsOptions in fields %}
               {% if fieldsOptions.component and not fieldsOptions.skip %}
-                {{ callAsMacro(fieldsOptions.component)(fieldsOptions) }}
```

Comment:
Do we want to disable autocomplete for all date inputs? If so then it could go here:
https://github.com/ministryofjustice/hmpps-locations-inside-prison/blob/90baa37f75e2417b404e700ecf01e626351979b2/server/controllers/base/formInitialStep.ts#L146-L168

If it's just for this page then lgtm

Reply:
The ticket only specifies this particular screen. It may be that additional tickets to remove autofill for all screens are needed in future. Will confirm that with UX and have those created if needed.